packages/docs-examples/components/badge/badge-list/badge-list-example.html

```diff
@@ -0,0 +1,28 @@
+<div class="badge-list-example">
+    <div class="badge-list-example__label kbq-extra-small-text">Horizontal</div>
+    <div class="badge-list-example__list">
+        <kbq-badge [badgeColor]="colors.FadeContrast">Bruteforce</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">Complex Attack</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">DDoS</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">DoS</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">HIPS Alert</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">IDS/IPS Alert</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">Identity Theft</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">Miscellaneous</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">Network Attack</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">Post Compromise</kbq-badge>
+    </div>
+    <div class="badge-list-example__label badge-list-example__label_vertical kbq-extra-small-text">Vertical</div>
+    <div class="badge-list-example__list badge-list-example__list_vertical">
+        <kbq-badge [badgeColor]="colors.FadeContrast">Bruteforce</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">Complex Attack</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">DDoS</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">DoS</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">HIPS Alert</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">IDS/IPS Alert</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">Identity Theft</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">Miscellaneous</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">Network Attack</kbq-badge>
+        <kbq-badge [badgeColor]="colors.FadeContrast">Post Compromise</kbq-badge>
+    </div>
```

Comment:
Здесь список названий одинаковый, можно вынести в массив

```suggestion
    <div class="badge-list-example__list">
        @for (badge of bages; track badge) {
        <kbq-badge [badgeColor]="colors.FadeContrast">{{ badge }}</kbq-badge>
        }
    </div>
    <div class="badge-list-example__label badge-list-example__label_vertical kbq-extra-small-text">Vertical</div>
    <div class="badge-list-example__list badge-list-example__list_vertical">
         @for (badge of bages; track badge) {
        <kbq-badge [badgeColor]="colors.FadeContrast">{{ badge }}</kbq-badge>
        }
    </div>
```